src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -15,6 +16,10 @@ internal class ManagedProjectSystemPackage : AsyncPackage
         public const long EditProjectFileCmdId = 0x1001;
         public const long GenerateNuGetPackageProjectContextMenuCmdId = 0x2000;
         public const long GenerateNuGetPackageTopLevelBuildCmdId = 0x2001;
+        public const string DefaultCapabilities = ProjectCapability.AppDesigner + "; " +
```

Comment:
Thanks, can you do some basic sanity testing for combined intellisense for cross targeting projects?